src/internal_macros.rs

```diff
@@ -240,7 +240,7 @@ macro_rules! serde_struct_impl {
             where
                 D: $crate::serde::de::Deserializer<'de>,
             {
-                use ::std::fmt::{self, Formatter};
+                use std::fmt::{self, Formatter};
```

Comment:
hmm I guess you're right, I didn't consider the fact that this only worked because `std` is used in the modules that use the macro :/ (1 more version bump and we could've removed all the `::` prefix hehe)

Reply:
W/o `::` prefix it will result in random build errors when the `use std` was removed from mods calling this macro to any external reason. Also, if at some point macro become exportable, it will break.

Reply:
I know, but from 1.30+ `::` prefix is never needed, but we're not there so I'd revert this

Reply:
Didn't know about 1.30, thanks for informing